test/lint/lint-python-dead-code-whitelist

```diff
@@ -12,6 +12,8 @@ InvalidOPIFConstruction  # unused class (test/functional/data/invalid_txs.py)
 _.is_compressed  # unused property (test/functional/test_framework/key.py)
 legacy  # unused variable (test/functional/test_framework/address.py)
 msg_generic  # unused class (test/functional/test_framework/messages.py)
+NODE_BLOOM  # actually used (test/functional/test_framework/messages.py)
+NODE_GETUTXO  # actually used (test/functional/test_framework/messages.py)
```

Comment:
This file was deleted in `master`. Mind to remove those changes again and force push? (No need to rebase, if you don't want to)